app/src/main/java/org/mozilla/focus/tips/TipManager.kt

```diff
@@ -49,11 +51,26 @@ class Tip(val id: Int, val text: String, val shouldDisplay: () -> Boolean, val d
             val id = R.string.tip_fresh_look
             val name = context.resources.getString(id, context.getString(R.string.app_name))
 
+            val sumoTopic = if (AppConstants.isKlarBuild)
+                SupportUtils.SumoTopic.WHATS_NEW_KLAR
+            else
+                SupportUtils.SumoTopic.WHATS_NEW_FOCUS
+
+            val whatsNewUrl = getSumoURLForTopic(context, sumoTopic)
+            val deepLink = {
+                context.components.tabsUseCases.addTab(
+                    url = whatsNewUrl,
+                    source = SessionState.Source.Internal.Menu,
+                    selectTab = true,
+                    private = true
+                )
+                TelemetryWrapper.pressTipEvent(id)
+            }
             val shouldDisplayFreshLookTip = {
                 Settings.getInstance(context).getAppLaunchCount() == 0
             }
 
-            return Tip(id, name, shouldDisplayFreshLookTip)
+            return Tip(id, name, shouldDisplayFreshLookTip,deepLink)
```

Comment:
This should fix the ktlint error 😄 

```suggestion
            return Tip(id, name, shouldDisplayFreshLookTip, deepLink)
```